test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2221,7 +2224,7 @@ public void testMetadataColumns_timeNull() throws Exception {
 
   @Test
   public void testMetadataColumns_timestamp() throws Exception {
-    executeUpdate("create table data(id integer, col timestamp)");
+    executeUpdate("create table data(id integer, col timestamp(3))");
```

Comment:
timestamp stores timestamps rounded to seconds.
timestamp(3) stores milliseconds.
timestamp(6) stores microseconds.